components/public/ItemCard.js

```diff
@@ -87,13 +90,14 @@ const ItemCard = ({ catalogItem = {}, showCartButton = true }) => {
             <Col sm="12" md="12">
               <Button
                 block
-                className={`float-right ${addingToCart && 'fade-btn'}`}
-                color={addingToCart ? 'success' : 'primary'}
+                className={`float-right ${addedToCart && 'fade-btn'}`}
```

Comment:
The loading indicator on the cards when items are displayed in a group wasn't working. This fixes that